src/renderer/modals/Receive/steps/StepReceiveFunds.js

```diff
@@ -47,6 +47,10 @@ const QRCodeWrapper = styled.div`
   background: white;
 `;
 
+const AlertBoxContainer: ThemedComponent<*> = styled.div`
```

Comment:
There seems to be a lint issue in the CI:

> Cannot resolve name `ThemedComponent`.

Reply:
fixed. wait for CI now